.yamllint

```diff
@@ -0,0 +1,39 @@
+yaml-files:
```

Comment:
I added a [lint task](https://github.com/defenseunicorns/uds-core/blob/b7a41e8acbcd9a5b717ba7d6304ec66cebd54dc5/tasks/lint.yaml#L8) that runs as yaml lint check and npx pepr format. This is the config file for the yaml lint. It is the same yam,l lint we use in [dubbd](https://github.com/defenseunicorns/uds-package-dubbd/blob/642e948ebdc67952d00ed6dfeb8088f326d9e863/.yamllint)

Reply:
fwiw i would have preferred this to function more like pepr format and actually do the formatting but it seemed like there were limited options for that for yaml specifically that wouldnt potentially cause more issues.

Reply:
sorry I meant, what is the `yamllint` binary? I don't have it on my system or see where we're installing it

Reply:
right. I dont install it because we dont install the tools in our run tasks per your request earlier on in this project (like uds and zarf install steps we originally had). yamllint can be installed via `pip install yamllint`. See dubbd's ci call: https://github.com/defenseunicorns/uds-package-dubbd/blob/642e948ebdc67952d00ed6dfeb8088f326d9e863/.github/workflows/ci-on-pull-request-k3d.yaml#L29

If you'd like i can add this as a CI step. I was initially adding so i could check formatting before pushing up changes as i said previously and wasnt trying to make this PR even larger by adding it to CI.

Reply:
added this as a ci step for it does a pepr format --check-validate and the yaml lint check in ci. Locally the task is still there to fix the formatting with pepr.

In the future i'd be happy to evaluate a different tool than yamllint especially if it can auto format in a non-breaking way, which so far the tools i've looked at either wont fix or cause issues with the zarf package schema once formatted.